include/bouncer.h

```diff
@@ -254,6 +266,10 @@ struct PgStats {
 	usec_t xact_time;	/* total transaction time in us */
 	usec_t query_time;	/* total query time in us */
 	usec_t wait_time;	/* total time clients had to wait */
+
```

Comment:
Maybe add a comment here like "prepared statements". Otherwise, it's not obvious what "ps" is.